docs/CleartextMessage.html

```diff
@@ -346,7 +346,7 @@ <h5>Parameters:</h5>
     
     <dt class="tag-source">Source:</dt>
     <dd class="tag-source"><ul class="dummy"><li>
-        <a href="https://github.com/openpgpjs/openpgpjs/blob/v5.3.1/src/cleartext.js">cleartext.js</a>, <a href="https://github.com/openpgpjs/openpgpjs/blob/v5.3.1/src/cleartext.js#L112">line 112</a>
+        <a href="https://github.com/openpgpjs/openpgpjs/blob/v5.3.1/src/cleartext.js">cleartext.js</a>, <a href="https://github.com/openpgpjs/openpgpjs/blob/v5.3.1/src/cleartext.js#L114">line 114</a>
```

Comment:
Could you un-commit the changes to the `docs` folder? That will make the diff easier to read, and they will be updated automatically on release anyway 😊

Reply:
Done!